.travis.yml

```diff
@@ -29,5 +29,4 @@ install:
   - if [ "$COMPOSER_PREFER" = "lowest" ]; then composer update --prefer-lowest -n; fi;
```

Comment:
oh... sorry, this line shouldn't have been merged previously... my bad

could you delete it please? (in order to have a "perfect" `v3.0.0`)

Reply:
The perfect v3 would be without atoum but that's already something  😛